### PR TITLE
lab03

### PR DESCRIPTION
**Pull request by crazyloofah**
lab03 company is complete. ONLY the home, about, and programs buttons work. I don't have any info on coaches, nor do I have an actual policy set in place. I'm trying to get this pushed (even if the code is sloppy because of time. Didn't try too hard to put in halfway decent text either, just filler text that might look somewhat real

### CODE REVIEW

File: code/matt/html_css_flask/lab03_company/lab03_company_page_1.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="test2.css" />
```
Comment (violette-git) [inline code/matt/html_css_flask/lab03_company/lab03_company_page_1.html:7]: Make sure you are linking to the correct stylesheet, and this one will be complete.

**Review by violette-git — COMMENTED** [review]
Make sure you are linking to the correct stylesheet, and this one will be complete.

**Comment by crazyloofah** [comment]
Stylesheet and links updated

**Review by violette-git — APPROVED** [review]
Looks good.

**Pull request merged**